backend/lookup/api/serializers.py

```diff
@@ -3,7 +3,7 @@
 
 from lookup.models import Visitor
 
-class UserSerializer(serializers.ModelSerializer):
+class VisitorSerializer(serializers.ModelSerializer):
```

Comment:
changed the serialiser's name to avoid ambiguity with the admin user's class name.